governance/internationals.md

```diff
@@ -0,0 +1,55 @@
+# International Bible Quizzing
+These rules apply to the Internationals quiz meet conducted once yearly, administered under the oversight of the CQLT.
+
+## Eligibility Requirements
+- All quizzers for International competition must be 12 to 18 years of age by the meet registration date or have been enrolled as a full-time high school student during the current quiz year.
```

Comment:
How about:

> a full-time high school student or equivalent

Reply:
I don't wanna touch eligibility. This is Zach territory IMO.

Reply:
defining high-school is tricky as different regions call them different things. Even in some areas of Michigan we call them different things. Example: 6th = Upper Elementary or Middle School or Secondary

It's also not unheard of for 12yr 6th graders...

Thoughts?

Reply:
I think the only objective thing would be to use age only, but I don't know if that runs you into any sort of problems with exceptional cases you want to exclude.

Reply:
19 yr old 12th graders are rare but not unheard of either. I guess we could say something like this...
>All International quizzers must be between the ages of 12 and 18. Any exceptions to this must be brought to the C&MA Quizzing Leadership Team (CQLT).

This matches the wording in the team structure section

Reply:
I think there are only 2 objective factors that you can use when defining eligibility:
(1) Age
(2) # of years quizzing

It would be very useful to list out the reasons why we care about limiting BQ to certain ages/grades/etc. Once you have that list, then you just write objective requirements. 

But right now we're just poking holes in existing and proposed language, but not really framing it against "is this how we want it?"

The concept of a "minor" is an interesting one, and one that is legally defined, no?

Reply:
I kinda don't think rewording eligibility should be part of v1. Thoughts?

Reply:
If the Super Squad can't figure out what the eligibility rules are, then they needs to be reworded in v1. I agree we shouldn't stray too far from FE, or ideally at all if possible. "Enrolled as a full-time high school student" is too vague IMHO.

Reply:
I don't disagree with the principle of "if can't figure out what the eligibility rules are, then they needs to be reworded in v1"

BUT, I think this is a whole potential mess, at the edges of the potential use cases.

99% of the time, this doesn't matter. 

BUT, to make the language actually objective and bulletproof, I think a lot of work needs to be done around "What is important regarding eligibility"--and that's not a v1 rewording problem. That's a thematic/goals-level problem.